test/utils/skip_test.py

```diff
@@ -0,0 +1,17 @@
+"""Skip specific tests."""
+import pytest
+
+
+def skip_adaptive_avg_pool3d_cuda(request) -> None:
+    """Skips test if AdaptiveAvgPool3d and cuda.
+
+    Args:
+        request: problem request
+    """
+    if all(
+        string in request.node.callspec.id for string in ["AdaptiveAvgPool3d", "cuda"]
+    ):
+        pytest.skip(
+            "Skip test because AdaptiveAvgPool3d does not work on cuda. "
+            "Is fixed in torch 1.9.0."
+        )
```

Comment:
Conditional skip based on PyTorch version?

Reply:
Does not seem to be fixed in torch 1.9.0. 
This is the commit in which it is fixed: https://github.com/pytorch/pytorch/commit/ddec2e0ef415feb9d72ed5105d3966d055151911
I'm not sure. Does this mean it is fixed in the next release (1.9.1)?

Reply:
Let's assume it will be in the next release, and skip if `torch<=1.9.0`

Reply:
1.9.0 is already released and it is not resolved. I assume 1.9.1.

Reply:
close https://github.com/f-dangel/backpack/issues/184 ?